ocs_ci/framework/conf/4_2_default_config.yaml

```diff
@@ -0,0 +1,40 @@
+---
+# This is the default configuration file for 4.2 version which will be merged
+#  with file passed
+# by:
+#
+# * --ocsci-conf file.yaml parameter.
+# * --cluster-conf cluster.yaml parameter - this will be rendered under
```

Comment:
We are deleting cluster conf in your other PR , please remove from here

Reply:
done